python/ray/actor.py

```diff
@@ -442,18 +448,26 @@ def _manual_init(self, *args, **kwargs):
             for k, v in self._ray_actor_methods.items():
                 self._actor_method_invokers[k] = ActorMethod(
                     k, self._ray_actor_id, self._ray_method_signatures[k])
+            
+            #Do not export the actor class or the actor if run in PYTHON_MODE
+            #Instead, instantiate the actor locally and add it to
+            #global_worker's dictionary
+            if ray.worker.global_worker.mode == ray.PYTHON_MODE:
+                ray.worker.global_worker.actors[self._ray_actor_id] = \
```

Comment:
To be more consistent with the rest of the code base, let's avoid using the `\` if possible. E.g., this could be formatted as

```python
ray.worker.global_worker.actors[self._ray_actor_id] = (
    Class.__new__(Class))
```

or something like that

Reply:
Yep, I can definitely change that.